addons/project/controllers/portal.py

```diff
@@ -94,7 +94,8 @@ def portal_my_project(self, project_id=None, access_token=None, **kw):
     @http.route("/my/project/<int:project_id>/project_sharing", type="http", auth="user", methods=['GET'])
     def render_project_backend_view(self, project_id):
         project = request.env['project.project'].sudo().browse(project_id)
-        if not project.exists() or not project.with_user(request.env.user)._check_project_sharing_access():
+
+        if not project.exists() and project.with_user(request.env.user)._check_project_sharing_access():
```

Comment:
Why has the second condition been reverted here?

Reply:
I revert my commit adding the access rights to portal user ('read', 'comment' and 'edit') because of the specs. 🙁 
By the way, I see the new condition is wrong, I will put the initial condition. Thanks